packages/api/src/router/server/server.ts

```diff
@@ -77,6 +78,13 @@ export const serverRouter = router({
 		);
 		return zServerPublic.parse(data);
 	}),
+	getAllServers: publicProcedure.query(async () => {
+		const data = await findOrThrowNotFound(
+			() => findAllServers(),
+			'No servers found',
+		);
+		return zServerPublic.parse(data);
```

Comment:
This will fail since it's parsing an array as a single object

Reply:
Fixed